nixos/modules/services/misc/matrix-synapse.nix

```
@@ -699,4 +699,11 @@ in {
       };
     };
   };
+
+  imports = [
+    (mkRemovedOptionModule ["services" "trusted_third_party_id_servers"] ''
+      Since #5835 gives the homeserver the ability to do both of these things itself, as well as the requirement for it to choose an external, trusted identity server if it so chooses, the homeserver no longer needs to constrain which identity servers are chosen.
```

What's `#5835`?

---

https://github.com/matrix-org/synapse/pull/5875 but I simplify the comment. Do you think it's better ?

---

Just use the release number. It should be enough to suggest the replacement option as in the [changelog](https://github.com/matrix-org/synapse/blob/v1.4.0/CHANGES.md#features).

```suggestion
      Since 1.4.0, "trust_identity_server_for_password_resets" is replaced by "account_threepid_delegates"
```

`account_threepid_delegates` should probably be added to the module.

---

Uhm, `trust_identity_server_for_password_resets` != `trusted_third_party_id_servers`. The first one should simply get renamed. Currently working on a fix.

---

Oops, copy/paste error of mine, indeed it's `trusted_third_party_id_servers` instead of `trust_identity_server_for_password_resets`.

---

Fixed in the current changeset :)